lighthouse-cli/printer.ts

```diff
@@ -129,7 +153,7 @@ function createOutput(results: Results, outputMode: OutputMode): string {
       item.subItems.forEach(subitem => {
         // Get audit object from inside of results.audits under name subitem.
         // Coming soon events are not located inside of results.audits.
-        subitem = results.audits[subitem] || subitem;
+        subitem = results.audits[subitem as any] || subitem;
```

Comment:
do we ever use this code path @paulirish? I feel like we should only index into audits if subitem is a string not always since we are going to be indexing in with `Object object` in a lot of cases 📦